rest-api-spec/src/main/resources/rest-api-spec/test/cluster.voting_config_exclusions/10_basic.yml

```
@@ -0,0 +1,85 @@
+teardown:
+  - do:
+      cluster.delete_voting_config_exclusions: {}
+
+---
+"Get cluster state without voting config exclusions":
+  - do:
+      cluster.state: {}
+
+  - length: { metadata.cluster_coordination.voting_config_exclusions: 0 }
+
+---
+"Add voting config exclusion by unknonw node Id":
```

Unknown spelling ;)

```suggestion
"Add voting config exclusion by unknown node Id":
```

(typo is duplicated elsewhere too)

---

Ops. Fixed.